packages/fxa-settings/.storybook/design-guide/pages/Colors.tsx

```diff
@@ -71,7 +72,7 @@ const Colors = ({ config }) => {
         <Copiable value="tablet:focus:bg-violet-300">
           <code className="mt-2 inline-block text-sm mr-2">
             <a
-              href="#"
+              href="#top"
```

Comment:
Not really worth changing either but I'm wondering if these should technically be buttons instead of links since `#top` doesn't mean anything? I can see why the linter complained about this being empty.

Reply:
I started to make them buttons, but it seemed like more effort / more likely to mess with styling. I can change them to buttons or file a follow up, let me know what you think.

Reply:
@LZoog I filed [FXA-7498](https://mozilla-hub.atlassian.net/browse/FXA-7498) to keep track of this.